Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.12.3-alpine3.19
```

Comment:
Any particular reason you are using python 3.12.3 here and another version in workflow?

Reply:
Missed that change to use similar versions in Dockerfile and github actions.